### PR TITLE
fixed broken links

### PR DESCRIPTION
**Pull request by michaelcretzman**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -833,7 +833,7 @@ The pipelines will run just fine, as you used the template and specified differe
 
 #### Service showing as active but hasn't been part of a deployment in over 30 days
 
-Harness shows the Active instances is say you had a deployment and the VM got deployed from a Harness deployment. No matter if we deploy anything else on the VM , until the VM is up and running as it is linked with the service. It will show as active instance. The 30 days mentioned [here](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd/#active-services) , is for service based licence calculation and usage for CD. 
+Harness shows the Active instances is say you had a deployment and the VM got deployed from a Harness deployment. No matter if we deploy anything else on the VM , until the VM is up and running as it is linked with the service. It will show as active instance. The 30 days mentioned [here](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd-for-cd/#active-services) , is for service based licence calculation and usage for CD.
```
Comment (aimurphy) [inline kb/continuous-delivery/continuous-delivery-faqs.md:836]: Just FYI, it says "-for-cd" twice.

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -2838,7 +2838,7 @@ We calculate the total number of instances that the task spawns, including sidec
 
 #### How do we detect service licenses for SSH deployments ?
 
-Please consider the following [Documentation](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd/#ssh-and-winrm).
+Please consider the following [Documentation](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd-for-cd/#ssh-and-winrm).
```
Comment (aimurphy) [inline kb/continuous-delivery/continuous-delivery-faqs.md:2841]: also for-cd twice

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ We can calculate the service licenses and instances in following methods for CG
 - Based on service instances we calculate the number of consumed licenses
 - 1 service license is equal to 20 active service instances
 
-Please find an example [here](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd/#example)
+Please find an example [here](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd-for-cd/#example)
```
Comment (aimurphy) [inline kb/continuous-integration/continuous-integration-faqs.md:54]: duplicate for-cd

File: release-notes/self-managed-enterprise-edition.md [hunk 4]
```diff
@@ -3511,7 +3511,7 @@ gsutil -m cp \
 
 - CD license utilization data was not reported for some accounts. (CDS-69101)
 
-  [License usage](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd/) was not retrieving the required information because the query to retrieve the license usage exceeded the connection timeout.
+  [License usage](https://developer.harness.io/docs/continuous-delivery/get-started/service-licensing-for-cd-for-cd/) was not retrieving the required information because the query to retrieve the license usage exceeded the connection timeout.
```
Comment (aimurphy) [inline release-notes/self-managed-enterprise-edition.md:3514]: duplicate for-cd

**Review by ravilach — APPROVED** [review]

**Pull request merged**